src/components/comments/CommentForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useState, useRef } from "react"
+import { Button, Form } from "react-bootstrap"
+import { CommentContext } from "./CommentProvider"
+
+
+export default ({postId}) => {
+    // the delcaration of content value for the state comment object is unnecessary at the current moment but will possibly
+    // be useful soon for default text
+    const [comment, setComment] = useState({ content: "" })
+    const { createComment } = useContext(CommentContext)
+    const commentRef = useRef("")
```

Comment:
seems like we could get rid of this ref? and therefore also the `useRef` import at the top and also the assigning of the ref to the actual `<Form.Control>` component, since we don't appear to be using it at all?